scenes/enumerated/problem_12649/problem_12649.py

```diff
@@ -0,0 +1,448 @@
+
+        self.play(FadeOut(VGroup(rect2, second_rule)))
+        self.wait(0.1)
+        self.play(
+            Indicate(miandam[7:12], scale_factor=SCALE_FACTOR),
+            miandam[12:14].animate.set_color(WHITE),
+            miandam[:6].animate.set_color(WHITE)
+            )
+        self.wait(0.25)
+        self.play(Write(third_rule), Create(rect3))
+        self.wait(0.25)
+        self.play(Write(miandam[14:]))
+        self.wait(0.25)
+
+        # օգտվենք 3-րդից
+        self.wait(0.25)
+        self.play(
+            Indicate(formulas[3][5], 1.5),
+            Indicate(formulas[3][9], 1.5)
+        )
+        self.wait(0.1)
+
+        self.play(
+            ModifyFormula(formulas[3], remove_items=[5,9], replace_items=[[7]], replace_items_strs=[['$^6$']]), 
+            )
+        self.wait(0.25)
+
+        self.play(Uncreate(VGroup(rect3, third_rule)), FadeOut(miandam))     
+        self.wait(0.25)
+
+        ans = Tex(' $=$ ', '$2$', '$x$', '$y$', r'$\cdot$', '$x$', '$^6$', '$y$', '$^3$', ' $+$ ', '$2$', '$x$', '$y$', r'$\cdot$', '$3$', '$x$', '$y$', font_size=FONT_SIZE)
+        ans.next_to(formulas[3], RIGHT)
+
+        self.play(
+            Indicate(formulas[3][4], 1.5),
+            Indicate(formulas[3][13], 1.5)
+        )
+        self.wait(0.25)
+
+        self.play(Write(ans[0]))
+        self.wait(0.1)
+
+        # տանում ենք a-ն ու b-ն
+        a_copy1 = formulas[3][0:3].copy()
+        b_copy = formulas[3][5:9].copy()
+        self.play(Wiggle(formulas[3][5:9], scale_value=WIGGLE_SCALE_FACTOR), Wiggle(formulas[3][0:3], scale_value=WIGGLE_SCALE_FACTOR))
+        self.wait(0.1)
+        self.play(
+            AnimationGroup(
+                CounterclockwiseTransform(a_copy1, ans[1:4], path_arc=PI/2),
+                FadeIn(ans[4]),
+                lag_ratio=0.5
+                ),
+            CounterclockwiseTransform(b_copy, ans[5:9], path_arc=PI/2)
+            )
+        self.add(ans[1:4]).remove(a_copy1)
+        self.add(ans[5:9]).remove(b_copy)
+        self.wait(0.25)
+
+        self.play(FadeIn(ans[9]))
+        # տանում ենք a-ն ու c-ն
+        a_copy2 = formulas[3][0:3].copy()
+        c_copy = formulas[3][10:13].copy()
+        self.play(Wiggle(formulas[3][10:13], scale_value=WIGGLE_SCALE_FACTOR), Wiggle(formulas[3][0:3], scale_value=WIGGLE_SCALE_FACTOR))
+        self.wait(0.1)
+        self.play(
+            AnimationGroup(
+                CounterclockwiseTransform(a_copy2, ans[10:13], path_arc=PI/2),
+                FadeIn(ans[13]), 
+                lag_ratio=0.5
+                ),
+            CounterclockwiseTransform(c_copy, ans[14:], path_arc=PI/2)
+            )
+        self.add(ans[10:13]).remove(a_copy2)
+        self.add(ans[14:]).remove(c_copy)
+        self.wait(0.25)
+
+        # = նշանը 14-րդ ինդեքսում ա
+        self.fix_formula(ans)
+        self.play(Wiggle(ans[1:9], scale_value=WIGGLE_SCALE_FACTOR))
+        seq = [14,15,16,19,20,17,18,21,22,23,24,25,26,27,28,29,30]
```

Comment:
If you feel like there is a need to add a comment here, please, do

Reply:
What comment?

Reply:
# The new formula is ....